### PR TITLE
Rd 365 add collaborators

### PR DESCRIPTION
**Pull request by dkosowski87**
## Changes
Adding possibility to add external users as collaborators to organisation repos.
Users marked `external/user.name` in team files will now be added as outside collaborators to team repos with permission stated in the team file.

### CODE REVIEW

File: app/models/github_integration/actions/sync.rb [hunk 1]
```diff
@@ -63,6 +72,7 @@ def create_teams(teams_to_create)
       end
 
       def new_team_add_members(members, team)
+        return if members.blank?
```
Comment (lubieniebieski) [inline app/models/github_integration/actions/sync.rb]: `members` looks like an array. Shouldn't we use `.any?` here?

File: app/models/github_integration/team_diff.rb [hunk 2]
```diff
@@ -64,17 +66,39 @@ def repos_diff(team, repos_names)
       end
     end
 
+    def collaborators_diff
+      expected_collaborators = map_users_to_collaborators
+      return if expected_collaborators.blank?
```
Comment (lubieniebieski) [inline app/models/github_integration/team_diff.rb]: similiar to my previous comment `.none?`?

File: app/models/github_integration/actions/sync.rb (hunk 1 above)
Comment (dkosowski87) [inline app/models/github_integration/actions/sync.rb]: To be on the safe side that perhaps members will be `nil` I used `blank?` it basically will return `true` if `members` is a `[]` or `nil`

File: app/models/github_integration/team_diff.rb (hunk 2 above)
Comment (dkosowski87) [inline app/models/github_integration/team_diff.rb]: same as in my above comment

**Pull request merged**